src/bioregistry/data/bioregistry.json

```diff
@@ -104533,6 +104460,79 @@
       "uri_format": "http://modelseed.org/biochem/reactions/$1"
     }
   },
+  "seed.subsystems": {
+    "biocontext": {
+      "prefix": "SEED.SUBSYSTEMS"
+    },
+    "contributor_extras": [
+      {
+        "email": "aireland@lbl.gov",
+        "github": "ialarmedalien",
+        "name": "AJ Ireland",
+        "orcid": "0000-0003-1982-9065"
+      },
+      {
+        "email": "seaver@anl.gov",
+        "github": "samseaver",
+        "name": "Sam Seaver",
+        "orcid": "0000-0002-7674-5194"
+      }
+    ],
+    "go": {
+      "description": "Project to annotate the first 1000 sequenced genomes, develop detailed metabolic reconstructions, and construct the corresponding stoichiometric matrices",
+      "homepage": "http://www.theseed.org",
+      "name": "The SEED;",
+      "prefix": "SEED.SUBSYSTEMS",
+      "uri_format": "http://www.theseed.org/linkin.cgi?id=$1"
+    },
+    "mappings": {
+      "biocontext": "SEED.SUBSYSTEMS",
+      "go": "SEED.SUBSYSTEMS",
+      "miriam": "seed.subsystems",
+      "n2t": "seed.subsystems",
+      "ncbi": "SEED.SUBSYSTEMS"
```

Comment:
Same here: all the mapped values in these external registries should remain unchanged.